### ISSUE TITLE
Midterm report peer review - Yu (Nancy) Gu yg234

### ISSUE THREAD

**Issue by nancy15689** [haoqiyao/NASDAQ-Index-Prediction#9]
This project aims to predict if the NASDAQ index will go up or down by using financial data of the previous day. It incorporates a wild range of data, including risk-free rates, exchange rates, commodity prices, different indices, and different topic's google trend data, which generate 20 features for the prediction. For preliminary analysis, a classification tree and linear logistic regression model are fit to 3000 training data points, and the test result indicates that both of these models are predicting the test data with 54 - 58% accuracy. 

Three things that the group did well: 
1. Very thorough analysis of data. Different techniques we learned from class to clean up messy data such as data inspection, changing null values, and removing outliers are used. The graphs generated are quite amazing, especially the heatmap which makes it very clear what columns are highly correlated and can be transformed by PCA. 
2. As baseline models, both linear and nonlinear models preform well. Although <60% accuracy is not good for final result since it is only slightly better than tossing a coin, it seems a reasonable baseline model to compare the final model on and test which possible improvements listed can affect the final model the most. 
3. A reasonably defined problem. Instead of trying to predict how much NASDAQ can go up or down or even a broader range of funds, the project starts with a simple binary classification with a proper amount of data on one single fund. This makes the project much easier and doable during a semester, and there is an appropriate amount of work in the future development section for the rest of the semester. 

Three things to improve: 
1. I am not sure if the financial data of the previous day can form good prediction of the next day's NASDAQ trend. A time series approach such as looking at data for the previous week might make better predictions since only a day worth of data might not capture some of the important trends happening in each feature which a time series approach can. 
2. Although some graphs such as the heatmap and histogram of NASDAQ are easy to understand, the others are hard without proper explanation. For example, the 5 feature distribution graphs can be easier to read if you have some explanation of which features seem to affect NASDAQ more. Although putting them in different graphs seems to show data better, it is harder to link them back to NASDAQ since it only appeares in one graph. 
3. Not many techniques are specified for preventing overfitting and underfitting. One of the requirements for midterm proposal is to come up with plans to avoid overfitting and underfitting, and even though the group mentioned that they should come up with more techniques in the future development section, it is actually a task that should have been completed.
